finality-aleph/src/nodes/mod.rs

```diff
@@ -15,9 +16,9 @@ use crate::{
     justification::{
         JustificationHandler, JustificationRequestSchedulerImpl, SessionInfo, SessionInfoProvider,
     },
-    last_block_of_session, mpsc,
+    mpsc,
     mpsc::UnboundedSender,
-    session_id_from_block_num,
+    session::SessionInfo as SessionBoundInfo,
```

Comment:
This suggests that perhaps this struct should be called `SessionBoundaryInfo` or something, since `SessionInfo` is a bit too generic and results in confusion with different session information here.

Reply:
Yeeeaaah, this could definitely be done. Will do.